crdb-seed/Cargo.toml

```diff
@@ -5,12 +5,15 @@ edition = "2021"
 license = "MPL-2.0"
 
 [dependencies]
+atomicwrites.workspace = true
+anyhow.workspace = true
 camino.workspace = true
 camino-tempfile.workspace = true
 dropshot.workspace = true
 hex.workspace = true
 omicron-test-utils.workspace = true
 ring.workspace = true
 slog.workspace = true
+tar.workspace = true
```

Comment:
Same here, I'm only seeing this used by `test-utils`

Reply:
Same.